shuup/campaigns/models/campaigns.py

```diff
@@ -118,6 +118,12 @@ class CatalogCampaign(Campaign):
     def __str__(self):
         return force_text(_("Catalog Campaign: %(name)s" % dict(name=self.name)))
 
+    def save(self, *args, **kwargs):
+        super(CatalogCampaign, self).save(*args, **kwargs)
+        if not self.active:
+            self.filters.update(active=self.active)
+            self.conditions.update(active=self.active)
```

Comment:
Hmm yes maybe better to just always update the `active` to be same with the campaign.